lib/offsite_payments/integrations/universal.rb

```diff
@@ -78,7 +78,8 @@ def sign_fields
         end
 
         def generate_signature
-          Universal.sign(@fields, @key)
+          fields_to_sign = @fields.select { |key, _| key.start_with?('x_') && key != 'x_signature' }
```

Comment:
The gem expects strings but I think but I could do `key.to_s.start_with?('x_')` to cover symbols. Thoughts?